### DISCUSSION TITLE
Azure Functions V4 and .net Framework 4.8 worker process support end date (the build requries .net 6)

### DISCUSSION THREAD

**Discussion by vlakshmi2024** [Azure/azure-functions-dotnet-worker#2355] [Feedback]
As per https://learn.microsoft.com/en-us/azure/azure-functions/dotnet-isolated-process-guide?tabs=windows the Azure Functio v4 with .isolated worker process targeting .net framework 4.8 requires .net 6 for Build Purposes.
Can someone confirm the end of support date for this? 
Will this be dependent on .net End of support date which is November 12, 2024?

**Comment by fabiocav** [comment]
@vlakshmi2024 there's no plan to end support for .NET Framework with isolated. That build dependency will be removed prior to .NET 6 EOL.

**Comment by jleibert** [comment]
Sorry if this is answered above, I am still new to azure functions.  We are using the .net framework 4.8 and when it deploys from visual studio it shows up in azure as .net 6.  The function runs under .net 8 however every time I redeploy it goes back to .net 6.  Is there a way to force visual studio to deploy to .net 8 or is the above saying that option will automatically happen in the future?

**Comment by skyblackhawk** [comment]
Hi to all,
    excuse me if I write here, but I can't open a new discussion. I suppose haven't right to do it.
So please Admin can move this discussion in a new discussion in correct container ?
Other thing, why in this github aren't discussions on new feature proposed by azure devops programmers ?

Title: New feature Azure function or it works? rx python library (reactive programming) and async with reactive I/O service developing an Azure function in cloud
I need to **increase performance** and I was wondering if a **devops user** as us **can write an Azure function, an API, in Python in reactive programming and async as all other services** as Database, file, App container,... in programming reactive and async.

Since I haven't found any type of discussion around, it can be done but no one has ever done it or it can't be done, could someone who has developed the Azure functions as a component please answer me?

We are talking about **multi-threads and asynchronicity programming of apps or services (DB, external APIs, disk I/O) etc etc etc program developer part** where the **thread goes to sleep and get thread that do not require I/O operations to proceed because done to get result which would greatly increase the performance of the Azure functions API on the cloud.**

Thanks a lot for your support, kind regards
